README.md

```diff
@@ -168,6 +168,7 @@ Hello Kyle
 ### Examples tools using Commander
 
 - [QueryKit](https://github.com/QueryKit/querykit-cli) via CocoaPods Rome
+- [SwiftGen](https://github.com/SwiftGen/SwiftGen)
```

Comment:
You could add `RasterizeXCAssets` also if you want to 

```suggestion
- [SwiftGen](https://github.com/SwiftGen/SwiftGen)
- [RasterizeXCAssets](https://github.com/LinusU/RasterizeXCAssets)
```

Would submit a PR myself but it would just conflict with yours 😅